### PR TITLE
Multipart Upload

### PR DESCRIPTION
**Pull request by sumandari**
Background:
Now we have an uploader class to upload a file to the API server. When the file has a large file size e.g. 10GB, there is a probability that the user will have an unpleasant experience when their connection is interrupted. 

Proposed solution:
Enable multipart upload by splitting a file into a chunk and sending them in separate API calls

### CODE REVIEW

File: src/mediacatch_s2t/uploader.py [hunk 1]
```diff
@@ -55,15 +78,17 @@ def _transcript_link(self):
     @staticmethod
     def _ffprobe(file_path) -> FFProbeResult:
         command_array = ["ffprobe",
-                        "-v", "quiet",
-                        "-print_format", "json",
-                        "-show_format",
-                        "-show_streams",
-                        file_path]
-        result = subprocess.run(command_array, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True)
+                         "-v", "quiet",
```
Comment (sumandari) [inline src/mediacatch_s2t/uploader.py:81]: This is a linter update. Nothing is changed in the code.
Automatically done by pycharm :grimacing:

File: src/mediacatch_s2t/uploader.py [hunk 2]
```diff
@@ -172,11 +206,215 @@ def upload_file(self):
...
+            "estimated_processing_time": self.estimated_result_time(
+                file_duration),
+            "message": "The file has been uploaded."
+        }
+        return self.result
+
+
 def upload_and_get_transcription(file, api_key, language):
+    is_multipart_upload = UploaderBase(
+        file, api_key, language).is_multipart_upload()
```
Comment (sumandari) [inline src/mediacatch_s2t/uploader.py:417]: It makes the uploader choose the upload method by itself

File: src/mediacatch_s2t/uploader.py [hunk 3]
```diff
@@ -31,6 +47,13 @@ def __init__(self, file, api_key, language='da'):
     def _is_file_exist(self):
         return pathlib.Path(self.file).is_file()
 
+    def is_multipart_upload(self) -> bool:
+        if self._is_file_exist():
+            filesize = os.path.getsize(self.file)
+            if filesize > CHUNK_SIZE_MIN:
```
Comment (sumandari) [inline src/mediacatch_s2t/uploader.py:53]: Please see: https://docs.aws.amazon.com/en_us/AmazonS3/latest/userguide/qfacts.html

There are requirements for multipart uploads

**Comment by FredHaa** [comment]
LGTM

**Review by FredHaa — APPROVED** [review]

**Pull request merged**